services/offline/src/lib/__tests__/online-status.test.js

```diff
@@ -0,0 +1,117 @@
+import { act, renderHook } from '@testing-library/react-hooks'
```

Comment:
Tests `useOnlineStatus` - initializing to `navigator.onLine` value, changing in response to `offline`/`online` events on `window`, and debouncing